common/djangoapps/course_modes/admin.py

```diff
@@ -189,6 +195,7 @@ class CourseModeAdmin(admin.ModelAdmin):
         'min_price',
         'currency',
         '_expiration_datetime',
+        'expiration_datetime_is_explicit',
```

Comment:
I find the use of "explicit" here a little vague.  I'm interested if other feel the same way, but something like `override_expiration_datetime` feels more descriptive.

Reply:
> I worry that to Studio users there is not feedback that this setting is in place and will feel like "spooky action at a distance." Everything updates except my upgrade deadline!!

@e0d , I can understand that.  We were kind of in the opposite position; users were updating their courses and noticing their upgrade_deadlines were being set to 10 days prior to the course end date even after they specified the deadline in the Django Admin interface.  It was difficult to track down what was triggering this.

Reply:
> I find the use of "explicit" here a little vague. I'm interested if other feel the same way, but something like `override_expiration_datetime` feels more descriptive.

I was just following the naming convention of the existing model attribute.  Should I update just the Admin screen to show `override_expiration_datetime` or should I update it throughout the model as well?